plugins/Jobs.cpp

```diff
@@ -367,8 +372,9 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
         // Update the data
         if (!db.write("UPDATE jobs "
                       "SET data=" +
-                      SQ(request["data"]) + " "
-                                            "WHERE jobID=" +
+                      SQ(request["data"]) + " " +
+                      (request.isSet("repeat") ? ", repeat=" + SQ(SToUpper(request["repeat"])) : "") +
```

Comment:
Oh, shit, sorry, misread the parenthesis. Forget me, I must probably be still drunk.